app/assets/stylesheets/custom.css.scss

```diff
@@ -160,6 +160,13 @@ h6 {
   background: #428bca;
 }
 
+// flash messages
+.flash {
+  a {
+    color: #ffffff;
```

Comment:
Color literals like `#ffffff` should only be used in variable declarations; they should be referred to via variable everywhere else.